skbuild/setuptools_wrap.py

```diff
@@ -459,6 +464,9 @@ def setup(*args, **kw):  # noqa: C901
     # weight and when CMake is given multiple times a argument, only the last
     # one is considered, let's prepend the one provided in the setup call.
     cmake_args = skbuild_kw['cmake_args'] + cmake_args
+    
+    # Used to confirm that the cmake executable is the same
+    cmake_cmd = [which('cmake')] + cmake_args
```

Comment:
I still think we should ask cmake what is its version.

After doing `pip  install -U cmake` or updating the version on the system, the path would be the same